### PR TITLE
[refactor] posix.mak: Enforce whitespace before opening parenthesis

### PR DESCRIPTION
**Pull request by ibuclaw**
Style rule taken from phobos posix.mak.

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -370,6 +370,9 @@ style_lint:
 	@echo "Check for trailing whitespace"
 	$(GREP) -nr '[[:blank:]]$$' $(MANIFEST) ; test $$? -eq 1
 
+	@echo "Enforce whitespace before opening parenthesis"
+	grep -nrE "\<(for|foreach|foreach_reverse|if|while|switch|catch)\(" $$(find src -name '*.d') ; test $$? -eq 1
```
[the remark this answers is not in the document]
Comment (ibuclaw) [reply posix.mak]: Not since object.di.  They should all be auto-generated.

File: posix.mak [hunk 2]
```diff
@@ -370,6 +370,9 @@ style_lint:
...
+
```
Comment (ZombineDev) [inline posix.mak]: Can you add a check for ~`version ( abc )`~ -> `version (abc)`

Comment (ibuclaw) [reply posix.mak]: I have it in a separate branch, do you really want it all squashed together?  This change alone is big enough as it is.

Comment (ibuclaw) [reply posix.mak]: There's loads more styling checks that would be nice to copy from phobos as well.  But I'm not going to spend too much time on that.

Comment (jacob-carlborg) [reply posix.mak]: > do you really want it all squashed together?

No.

**Pull request merged**